api/v1/sriovnetworknodestate_types.go

```diff
@@ -119,6 +119,9 @@ type System struct {
 	// +kubebuilder:validation:Enum=shared;exclusive
 	//RDMA subsystem. Allowed value "shared", "exclusive".
 	RdmaMode string `json:"rdmaMode,omitempty"`
+	// OVS config. It will be provided for ovs-vswitchd service as other_config option
+	// +kubebuilder:default:= "hw-offload=true"
+	OvsConfig string `json:"ovsConfig,omitempty"`
```

Comment:
Can we use map[string]string here?

Reply:
I would like to support all available options, so this syntax will allow us to easly provide options like `other-config:mtu!=1500`